src/components/NavMenu.tsx

```diff
@@ -45,19 +45,23 @@ export const NavMenu = () => {
     <div className="absolute bottom-4 left-1/2 z-50 flex -translate-x-1/2 gap-x-2">
       <ul className="menu rounded-box menu-horizontal bg-base-200">
         {LINKS.map((link) => (
-          <li
-            key={link.href}
-            className={cn("", {
-              "text-accent": activePage === link.href.slice(1),
-            })}
-          >
-            <Link href={link.href}>{link.icon}</Link>
+          <li key={link.href} className="bg-transparent">
+            <div>
+              <Link
+                href={link.href}
+                className={cn({
+                  "text-accent": activePage === link.href.slice(1),
+                })}
+              >
+                {link.icon}
+              </Link>
+            </div>
           </li>
         ))}
       </ul>
       <div className="tooltip" data-tip="Logout">
         <button
-          className={cn("btn btn-square", "rounded-2xl")}
+          className="btn btn-square rounded-2xl"
```

Comment:
There's no difference in this case between using cn and without it.

Reply:
I'm okay with making this change for now, but I worry that this might be a point of failure in the future if one of the `btn` classes conflicts with the `rounded-2xl` tw class.